src/pg/sql/11_markov.sql

```diff
@@ -0,0 +1,81 @@
+-- Spatial Markov
+
+-- input table format:
+-- id | geom | date_1 | date_2 | date_3
+--  1 | Pt1  | 12.3   | 13.1   | 14.2
+--  2 | Pt2  | 11.0   | 13.2   | 12.5
+--  ...
+-- Sample Function call:
+-- SELECT CDB_SpatialMarkov('SELECT * FROM real_estate',
+--                          Array['date_1', 'date_2', 'date_3'])
+
+CREATE OR REPLACE FUNCTION
+  CDB_SpatialMarkovTrend (
+      subquery TEXT,
+      time_cols TEXT[],
```

Comment:
We could try using the [pyagg](https://github.com/CartoDB/crankshaft/blob/068c80f369dec4280cb1b6932aa6d57691836bbe/doc/04_pyAgg.md#pyagg-helper-function)  helper I wrote for Segmentation to collect and pass in the time columns and the data columns? Probably good to go as is but we could perhaps try and implement that as another function signature.